benches/src/iter.rs

```diff
@@ -0,0 +1,95 @@
+use std::time::Duration;
+
```

Comment:
authoring a benchmark I'd expect those to be on those on `BenchmarkContext`'s `impl`.
Also, some doc string on when to use what would be good for that